x-pack/plugins/index_management/public/sections/index_list/components/index_actions_context_menu/index_actions_context_menu.js

```diff
@@ -302,11 +300,9 @@ class IndexActionsContextMenuUi extends Component {
             <p>
               <FormattedMessage
                 id="xpack.idxMgmt.indexActionsMenu.forceMerge.forceMergeDescription"
-                defaultMessage="You are about to force merge {oneIndexSelected, plural, one {this} other {these}}"
-                values={{ oneIndexSelected: oneIndexSelected ? 1 : 0 }}
+                defaultMessage="You are about to force merge {selectedIndexCount, plural, one {this index:} other {these indices:}}"
```

Comment:
I think the colon `:` can be extracted outside of the pluralization placeholder.